src/lib/holocene/modal.stories.svelte

```diff
@@ -28,6 +28,7 @@
     confirmText: 'Confirm',
     cancelText: 'Cancel',
     confirmType: 'destructive',
+    open: true,
```

Comment:
I figured out why this isn't working. In the modal component, we have `$: open, toggleModal();`, and in toggleModal, we call `modalElement?.show()` if `open` is true, however `modalElement` is not defined yet. We could change that reactive statement to `$: (open, modalElement), toggleModal();`, however it's not really a true use case to mount the modal already opened, so I don't think we really need to.

Reply:
In that case maybe the better default is `false`. Will update!